synapse/models/inet.py

```diff
@@ -48,17 +46,28 @@ def getDataModel():
             ('inet:mac',  {'subof':'str', 'regex':'^([0-9a-f]{2}[:]){5}([0-9a-f]{2})$', 'lower':1, 'nullval':'??',
                            'ex':'aa:bb:cc:dd:ee:ff','doc':'A 48 bit mac address'}),
 
-            ('inet:netuser',  {'subof':'sepr','sep':'/','fields':'site,inet:fqdn|user,inet:user',
+            ('inet:netuser',{'subof':'sepr','sep':'/','fields':'site,inet:fqdn|user,inet:user',
                                'doc':'A user account at a given web address','ex':'twitter.com/invisig0th'}),
 
-            ('inet:netmesg',  {'subof':'sepr','sep':'/','fields':'site,inet:fqdn|from,inet:user|to,inet:user|sent,time',
-                               'doc':'A message sent from one user to another within a web community',
-                                'ex':'twitter.com/invisig0th/gobbles/20041012130220'}),
+            ('inet:netpost',{'subof':'sepr', 'sep':'@', 'fields':'netuser,inet:netuser|time,time',
+                             'doc':'A post made by a netuser at a specific time','ex':'twitter.com/invisig0th/20150302225401'}),
+
```

Comment:
shouldn't the example be "twitter.com/invisigoth@20150302225401" ?
Not all netposts may have high-resolution timestamps available, so we could have multiple posts from a single user which are truly unique in the realworld but in the model would appear as one.
If we had two invisigoth tweets and we only knew they were from 2015, how would we store those?